design/031-remove_compiler_base.md

```diff
@@ -0,0 +1,41 @@
+# Proposal: Remove "compiler.base" settings mechanism
+
+
+| **Status**        |                                                   |
+|:------------------|:--------------------------------------------------|
+| **RFC #**         | [031](https://github.com/conan-io/tribe/pull/31)  |
+| **Submitted**     | 2022-01-17                                        |
+| **Tribe votes**   |                                                   |
+
+
+## Summary
+
+The mechanism of defining that a compiler such as “intel” in the settings.yml definition will kind of extend other existing compilers (such as “intel” having as a base either “gcc” in Linux or “msvc” in Windows) will be removed. Instead, a full definition of the compiler will be necessary, as the new IntelOneApi (via ``intel-cc`` setting has already been proposed in Conan 1.X).
```

Comment:
I can't parse this sentence, it reads like it was half-rewritten and something remained there?

Reply:
Probably the parenthesis is wrong, would 
```
 Instead, a full definition of the compiler will be necessary, as the new IntelOneApi (via ``intel-cc`` setting) has already proposed in Conan 1.X.
```
be better?

Reply:
Oh sorry, I meant the first sentence:

```
The mechanism of defining that a compiler such as “intel” in the settings.yml definition will kind of extend other existing compilers (such as “intel” having as a base either “gcc” in Linux or “msvc” in Windows) will be removed.
```

Reply:
Got it, will rewrite it right now to clarify

Reply:
Done, please re-check https://github.com/conan-io/tribe/pull/31/commits/08a7c01e954f2924cd9f5797ee1f77c63377f407 proposal to see if more clear now.

Reply:
Yeah that's much better, thanks!